sdk/iot/Azure.Iot.Hub.Service/src/Customized/JobRestClient.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service
+{
+    internal partial class JobRestClient
+    {
+        internal Response<string> CancelImportExportJob(string id, CancellationToken cancellationToken = default)
+        {
+            if (id == null)
```

Comment:
Use String.IsNullOrWhitespace(id)

Reply:
On another note, this probably a copied class from the generated folder. So don't change this. We can add necessary validation in IotHubClient.